docs/admin-manual/monitoring.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Monitoring
+sidebar_position: 85
+---
+
+This page provide some guidance to monitor LibreTime.
+
+## Sentry
+
+To gather and remotely monitor exceptions that may occur in your installation, you can use the Sentry library shipped in LibreTime to send reports to any Sentry compatible server.
+
+To configure Sentry in LibreTime, you need to:
+
+- install the LibreTime Python packages with the `sentry` extra (the container images already ship the sentry extra),
+- set the `SENTRY_DSN` environment variable on each of the LibreTime services you want to monitor.
```

Comment:
Can you elaborate on what is expected to be in this environment variable?

Reply:
I added a like to the Sentry [Data Source Name](https://docs.sentry.io/product/sentry-basics/dsn-explainer/) docs.